app/forms/base_housing_costs_form.rb

```diff
@@ -35,27 +35,28 @@ def frequencies
 private
 
   def housing_benefit_does_not_exceed_costs
-    # Skip validation if housing benefit isn't relevant or value/frequency is missing
     return unless housing_benefit_relevant &&
-      housing_benefit_value.to_i.positive? &&
+      housing_benefit_value.respond_to?(:positive?) &&
```

Comment:
I'll be honest, I've never heard of `respond_to?`, but that is a useful ruby method. 

Not a blocker, but do you think we should change the checking of a number to something like this: 

`housing_benefit_value.is_a?(Numeric)`

I've just seen the use of `is_a?` more in CCQ.

Reply:
@willc-work did you also want to change it to `housing_benefit_value.is_a?(Numeric)` on this bit?

Reply:
yes thanks I have just updated it